sdk/clientcore/core/src/main/java/io/clientcore/core/utils/AuthUtils.java

```diff
@@ -1,10 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package io.clientcore.core.utils.auth;
+package io.clientcore.core.utils;
```

Comment:
We should make this AuthUtils type internal.

Reply:
It's being used by `http-okhttp3`, so will need to remain public for a moment.

I'm planning to review the usage in `http-okhttp3` in a separate PR as it appears to involve logic changes.